sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ReadmeSamples.java

```diff
@@ -341,7 +348,153 @@ public void databaseCreateContainerSample() {
             System.out.println("Failed to create container: " + ce);
         }
         // END: com.azure.cosmos.CosmosDatabase.createContainer
+    }
+
+
+    public void getDatabaseSample() {
+        CosmosAsyncClient cosmosAsyncClient = new CosmosClientBuilder()
+            .buildAsyncClient();
+        // BEGIN: com.azure.cosmos.CosmosAsyncClient.getDatabase
+        String databaseId = "<YOUR DATABASE NAME>";
+
+        CosmosAsyncDatabase database = cosmosAsyncClient.getDatabase(databaseId);
+        // END: com.azure.cosmos.CosmosAsyncClient.getDatabase
+    }
+
+    public void createGlobalThroughputControlConfigBuilderSample() {
+        CosmosAsyncClient cosmosAsyncClient = new CosmosClientBuilder()
+            .buildAsyncClient();
+        // BEGIN: com.azure.cosmos.CosmosAsyncClient.createGlobalThroughputControlConfigBuilder
+        String databaseId = "<YOUR DATABASE NAME>";
+        String containerId = "<YOUR CONTAINER NAME>";
+
+        GlobalThroughputControlConfig config =
+            cosmosAsyncClient.createGlobalThroughputControlConfigBuilder(databaseId, containerId)
+                .build();
+        // END: com.azure.cosmos.CosmosAsyncClient.createGlobalThroughputControlConfigBuilder
+    }
+
+    public void readConflictSample() {
+        // BEGIN: com.azure.cosmos.CosmosAsyncConflict.read
+        String conflictId = "CONFLICT_ID";
+        CosmosAsyncDatabase cosmosAsyncDatabase =
+            new CosmosAsyncDatabase(conflictId, cosmosAsyncClient);
+        CosmosAsyncContainer cosmosAsyncContainer =
+            new CosmosAsyncContainer(conflictId, cosmosAsyncDatabase);
+        CosmosConflictRequestOptions options =
+            new CosmosConflictRequestOptions();
+        CosmosAsyncConflict conflict =
+            new CosmosAsyncConflict(conflictId, cosmosAsyncContainer);
+
+        Mono<CosmosConflictResponse> conflictResponseMono = conflict.read(options);
+        // END: com.azure.cosmos.CosmosAsyncConflict.read
+    }
+
+    public void CosmosAsyncProcedureReadSample() {
+        CosmosAsyncDatabase cosmosAsyncDatabase = new CosmosAsyncDatabase(id, cosmosAsyncClient);
```

Comment:
Same here - /**
     * Gets a database object without making a service call.
     *
     * @param id name of the database.
     * @return {@link CosmosAsyncDatabase}.
     */
    public CosmosAsyncDatabase getDatabase(String id) {
        return new CosmosAsyncDatabase(id, this);
    }